sdk/elixir/runtime/main.go

```diff
@@ -2,6 +2,7 @@ package main
 
 import (
 	"context"
+	"dagger/runtime/internal/dagger"
```

Comment:
I just realize that I declare a `module` in go.mod as `dagger/runtime`. I think renaming it to `elixir-sdk` for consistent with Python. 

BTW, the TypeScript using `main` as a module name. Do we need to renaming it into `typescript-sdk`? I can open a PR to fix this if you want.

Reply:
Fixed the Elixir go module name. 👍